vcsutils/webhookparser/bitbucketserver_test.go

```diff
@@ -39,12 +41,16 @@ func TestBitbucketServerParseIncomingPushWebhook(t *testing.T) {
 
 	// Check values
 	assert.Equal(t, expectedRepoName, actual.TargetRepositoryDetails.Name)
-	assert.Equal(t, expectedOwner, actual.TargetRepositoryDetails.Owner)
+	assert.Equal(t, formatOwnerForBitbucketServer(expectedOwner), actual.TargetRepositoryDetails.Owner)
 	assert.Equal(t, expectedBranch, actual.TargetBranch)
 	assert.Equal(t, bitbucketServerPushExpectedTime, actual.Timestamp)
 	assert.Equal(t, vcsutils.Push, actual.Event)
 }
 
+func formatOwnerForBitbucketServer(owner string) string {
+	return fmt.Sprintf("~%s", strings.ToUpper(owner))
+}
```

Comment:
For the sake of order, please relocate this function below all tests in this file.

Reply:
done